frontend/src/components/auth/AuthNavbar.tsx

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+import { useHistory } from "react-router-dom";
+import {
+  Flex,
+  Box,
+  Button,
+  Menu,
+  MenuButton,
+  MenuList,
+  MenuItem,
+} from "@chakra-ui/react";
+import { ArrowBackIcon, ChevronDownIcon } from "@chakra-ui/icons";
+
+const AuthNavbar = (): React.ReactElement => {
+  const history = useHistory();
+
+  return (
+    <Box px="90px" boxShadow="md">
+      <Flex h="80px" alignItems="center" justifyContent="space-between">
+        <Button
+          leftIcon={<ArrowBackIcon w={8} h={8} />}
+          variant="link"
+          onClick={() => window.open("https://sistering.org/", "_blank")}
+        >
+          Back to Main
+        </Button>
```

Comment:
Unless @JoshuaYe said otherwise, I think opening the link in the current page would flow better (rather than open new tab)

Let's try to avoid `window.open` and use Chakra UI's built in link component

```suggestion
        <Link href="https://sistering.org/">
          <Button leftIcon={<ArrowBackIcon w={8} h={8} />} variant="link">
            Back to Main
          </Button>
        </Link>
```